src/functions-and-arrays.js

```diff
@@ -52,15 +110,36 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+function uniquifyArray(wordsUnique) {
+  if (wordsUnique.length === 0) {
+    return null;
+  }
+    let uniquifyWords = [];
+    for(let i of wordsUnique) {
+      if(uniquifyWords.indexOf(i) < 0) {
+        uniquifyWords.push(i);
+      }
+    }
+    return uniquifyWords;
+}
+console.log(uniquifyArray(wordsUnique))
 
 
 // Iteration #6: Find elements
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
-
+function doesWordExist(wordsFind, value) {
+  if (wordsFind.length === 0) {
+    return null;
+  }
+    let wordSeeker = false;
+    for(let i of wordsFind) {
+      if(i == value) {
+          wordSeeker = true;
+      }   
+    } 
+    return wordSeeker;  
+}
```

Comment:
Here, you can save yourself some typing by just returning true when it's true and false in all other cases (no need for an extra helper variable):
```
for (let el of ar) { 
  if (el === word)  return true;  
   return false;
```

Reply:
This one looks great. Thanks.


Also, yes, I am looking forward to doing the bonuses but I need to stay up to date with the course first and so far it is demanding my time.